mule-apikit-tools/mule-apikit-scaffolder/src/test/java/org/mule/tools/apikit/ScaffolderTest.java

```diff
@@ -71,6 +71,7 @@ public void testSimpleGenerate(String name) throws Exception {
     assertEquals(1, countOccurences(s, "<http:listener-config"));
     assertEquals(1, countOccurences(s, "get:\\:" + name + "-config"));
     assertEquals(1, countOccurences(s, "get:\\pet:" + name + "-config"));
+    assertEquals(1, countOccurences(s, "get:\\pet/v1:" + name + "-config"));
```

Comment:
the flow name should be "pet\\v1" instead of "pet/v1", because "\" is not a valid value  for flow names